src/server/cluster/cluster_family.cc

```diff
@@ -149,21 +149,24 @@ ClusterShardInfo ClusterFamily::GetEmulatedShardInfo(ConnectionContext* cntx) co
                                   ? static_cast<uint16_t>(absl::GetFlag(FLAGS_port))
                                   : cluster_announce_port;
 
-    info.master = {.id = id_, .ip = preferred_endpoint, .port = preferred_port};
+    info.master = {{.id = id_, .ip = preferred_endpoint, .port = preferred_port}, NodeHealth::NONE};
```

Comment:
why not online here?

Reply:
today we alway reply with online for all nodes in emulated mode, so unless we want to do any change there you should use online here